dev-requirements.txt

```diff
@@ -20,7 +20,8 @@ pyre-extensions
 pyre-check
 pytest
 pytest-cov
-pytorch-lightning==1.5.10
+pytorch-lightning
+tensorboard
```

Comment:
why do we need this?

Reply:
The latest pytorch-lightning needs that: 

<img width="1876" alt="image" src="https://github.com/pytorch/torchx/assets/13003249/85c3afa2-7268-45bd-901c-94ae2b5a056d">